README.md

```diff
@@ -1,34 +1,84 @@
 
-* unsupported for call signatures:
+## Tests and known issues
+srclib-typescript is alpha-quality software. It powers code analysis on
+[Sourcegraph.com](https://sourcegraph.com) and additionally has been tested on the next typescript repositories:  
 
-```typescript
-interface Foo1 {
-    bar: {
-        (s: string): string;
-        //(n: number): string;
-    }
-}
-var foo_inst: Foo1 = { bar(s: string): string { return "1"; } }
-```
+**Successfully:**
+* [n3-charts/line-chart](https://github.com/n3-charts/line-chart) 
+* [palantir/plottable](https://github.com/palantir/plottable)
+* [SierraSoftworks/Iridium](https://github.com/SierraSoftworks/Iridium)
+* [valor-software/ng2-bootstrap](https://github.com/valor-software/ng2-bootstrap) 
+* [DefinitelyTyped/tsd](https://github.com/DefinitelyTyped/tsd)
+* [Microsoft/code-push](https://github.com/Microsoft/code-push)
+* [Microsoft/TypeScriptSamples](https://github.com/Microsoft/TypeScriptSamples)
+* [mgechev/angular2-seed](https://github.com/mgechev/angular2-seed)
+* [Microsoft/vscode-go](https://github.com/Microsoft/vscode-go)
+* [ng-book/angular2-rxjs-chat](https://github.com/ng-book/angular2-rxjs-chat)
+* [auth0/angular2-authentication-sample](https://github.com/auth0/angular2-authentication-sample)
+* [NativeScript/nativescript-cli](https://github.com/NativeScript/nativescript-cli)
+* [uProxy/uproxy](https://github.com/uProxy/uproxy)
+* [rogerpadilla/angular2-minimalist-starter](https://github.com/rogerpadilla/angular2-minimalist-starter)
+* [rangle/batarangle](https://github.com/rangle/batarangle)
+* [VSCodeVim/Vim](https://github.com/VSCodeVim/Vim)
+* [ngUpgraders/ng-forward](https://github.com/ngUpgraders/ng-forward)
+* [vega/vega-lite](https://github.com/vega/vega-lite)   
 
-## Toolchain tested on:
-1. project itself - https://sourcegraph.com/antonina-cherednichenko/srclib-typescript
-2. line-chart - https://sourcegraph.com/n3-charts/line-chart
-3. sourcegraph tests - https://sourcegraph.com/sgtest/typescript-test
-4. MS TypeScript samples - https://sourcegraph.com/Microsoft/TypeScriptSamples
+**Testing in progress:**
+* [plasma-umass/doppio](https://github.com/plasma-umass/doppio) 
+* [DefinitelyTyped/DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped) 
 
+**Next issues were found:**
+* Unsupported builtins for interfaces.
+* Partially supported shorthand property assignments.
+* ExportSpecifiers can work better.
+* Import clause is not fully supported (syntax tree kind = 226, reproduced on [doppio](https://github.com/plasma-umass/doppio) project).
 
+## Run procedure
+Testing this toolchain requires that you have installed `src` from
+[srclib](https://sourcegraph.com/sourcegraph/srclib) and that you have this
+toolchain set up. See srclib documentation for more information.
+To test this toolchain's output against the expected output, run:
```

Comment:
`srclib` (former `src`) does not support `-m` flag anymore. Also, there is no need to build Docker image before running the tests